src/browser/connection_manager.ts

```diff
@@ -73,10 +73,19 @@ function isMeshEnabled(args: ArgMap) {
     return enabled;
 }
 
+function keyFromPortInfo(portInfo: PortInfo): string {
+    const { version, port, securityRealm } = portInfo;
+    return `${version}/${port}/${securityRealm ? securityRealm : ''}`;
+}
+
 if (multiRuntimeEnabled && isMeshEnabled(coreState.argo)) {
     startConnectionManager();
 }
 
+function getMeshUuid(): string {
+    const portInfo = <PortInfo>coreState.getSocketServerState();
```

Comment:
I might be wrong, but shouldn't it be `const portInfo: PortInfo = coreState.getSocketServerState();` or it's the same thing just different signature...?

Reply:
getSocketServer state is not typed.